queue/bytes_queue_test.go

```diff
@@ -344,10 +345,10 @@ func TestMaxSizeLimit(t *testing.T) {
 	queue.Push(blob('b', 5))
 	capacity := queue.Capacity()
 	_, err := queue.Push(blob('c', 15))
+	assertEqual(t, err, nil)
 
 	// then
 	assertEqual(t, 50, capacity)
-	assertEqual(t, "Full queue. Maximum size limit reached.", err.Error())
```

Comment:
We should bring back this assertion

Reply:
So I should insert one more entry to make get a full queue error?

Reply:
Yes please or you can extend existing blobs